doc/docker.md

```diff
@@ -55,224 +63,211 @@ b73d677399de:/# psql -U postgres -c "SHOW ALL" | grep maintenance
 ```
 
-`SHOW` displays the current setting of run-time parameters. Be aware that it can differ from the values in `postgresql.conf`.
-
-### To connect to another network:
+`SHOW` displays the current setting of run-time parameters. Be aware that it can differ
+from the values in `postgresql.conf`.
 
-To connect to different network (preprod or preview) use `NETWORK` environment variable:
+## Running `cardano-db-sync`
 
+Start `cardano-db-sync`:
 
-```
-NETWORK=preprod docker-compose up -d && docker-compose logs -f
+```bash
+docker run \
+  --env NETWORK=mainnet \
+  --env POSTGRES_HOST=postgres \
+  --env POSTGRES_PORT=5432 \
+  --volume db-sync-data:/var/lib/cexplorer \
+  --volume node-ipc \
+  ghcr.io/IntersectMBO/cardano-db-sync:13.3.0.0
 ```
 
-### Overriding Configuration
+### Environment Variables
 
-Overriding the configuration file can be done by passing the `DB_SYNC_CONFIG` environment
-variable. First, add a bind mount to the `cardano-db-sync` service:
-
-```yaml
-cardano-db-sync:
-  image: cardano-db-sync
-  # <-- Snip -->
-  volumes:
-    - db-sync-data:/var/lib/cexplorer
-    - node-ipc:/node-ipc
-    # Add the bind mount here
-    - ./config/network/mainnet:/data
-  # <-- Snip -->
-```
+#### `NETWORK`
 
-Set the configuration file with `DB_SYNC_CONFIG`:
+Specifies a network to connect to. If specified, `cardano-db-sync` will provide the
+configuration files. If not specified, will call the `cardano-db-sync` executable as the
+entrypoint.
 
-```sh
-export DB_SYNC_CONFIG=/data/db-sync-config.json
-docker-compose up -d && docker-compose logs -f
-```
+#### `POSTGRES_HOST` (required)
 
-### Take control
+The PostgreSQL server host to connect to.
 
-Excluding the `NETWORK` ENV will simply just call the `cardano-db-sync` executable
-as the entrypoint, so you must pass a command and
-arguments in this case to provide config at runtime. The `--schema-dir` argument is preset,
-so is not required.
+#### `POSTGRES_PORT` (required)
 
-Using Docker run to demonstrate, but can be achieved using `docker-compose` too via
-`service.command`
-```
-docker run \
-  -v $PWD/config/network/mainnet/cardano-db-sync:/config \
-  -v $PWD/config/network/mainnet/genesis:/genesis \
-  -v $PWD/node-ipc:/node-ipc \
-  ghcr.io/IntersectMBO/cardano-db-sync:13.1.1.2-docker \
-    run --config /config/config.yaml --socket-path /node-ipc/node.socket # command
-```
+Specifies the PostgreSQL server port to connect to.
 
-## Build and load image using Nix
+#### `RESTORE_SNAPSHOT` (optional)
 
-```
-nix build .#dockerImage
-docker load -i ./result
+Specifies a `cardano-db-sync` snapshot archive to download and restore from. If omitted,
+it will sync from genesis.
+
+### `DB_SYNC_CONFIG` (optional)
+
+Overrides the `db-sync-config.json` provided by the network configuration.
+
+#### `EXTRA_DB_SYNC_ARGS` (optional)
+
+Extra command line arguments to pass to `cardano-db-sync`.
```

Comment:
I'm not sure if you agree but would be cool to have little examples of each arg? Also maybe a little link to the command line args

Reply:
> maybe a link to the command line args docs

On second thought, we apparently don't have CLI args docs anymore :shrug:

Reply:
Fixed up a little bit, any better?

Reply:
perfect ❤️ 

> On second thought, we apparently don't have CLI args docs anymore 🤷

did we/I accidentally remove them?

Reply:
No, we turned it into the config docs

Reply:
ah yeah of course, I don't mind adding them in another PR so we get this PR through